### PR TITLE
add favicon

### PR DESCRIPTION
**Pull request by Alicia-isa**

### CODE REVIEW

File: views/layout.pug [hunk 1]
```diff
@@ -11,6 +11,7 @@ html(lang="en")
     meta(name='keywords', content='ethereum, explorer, blockchain, etherchain, cryptocurrency, cryptocurrencies, bitcoin')
 
     title ewasm testnet
+    link(rel='shortcut icon' href='/explorer/images/favicon.ico')
```
Comment (axic) [inline views/layout.pug]: Thanks! I think lately favicon's don't need to be this horrible tiny ICO format, but can be larger PNGs.

**Review by axic — CHANGES_REQUESTED** [review]
Please use a larger PNG icon.

**Comment by axic** [comment]
@hugo-dc could you address this?

**Comment by hugo-dc** [comment]
sure

**Review by axic — APPROVED** [review]

**Pull request merged**